src/Microsoft.Performance.Toolkit.Engine/Engine.cs

```diff
@@ -561,7 +470,7 @@ public RuntimeExecutionResults Process()
                 assemblyDiscovery,
                 repoBuilder);
 
-            assemblyDiscovery.ProcessAssemblies(extensionDirectory);
+            assemblyDiscovery.ProcessAssemblies(extensionDirectory, out _);
```

Comment:
> out _ [](start = 67, length = 6)

should we add a parameter to CreateCore so that callers can get this information?